_observability-plugin/trace/trace-analytics-jaeger.md

```diff
@@ -0,0 +1,249 @@
+
+``` 
+docker compose down
+```
+
+### Step 3: Generate sample data
+
+Use the sample app provided with the Docker file to generate data. After you run the Docker compose file, it runs the sample app in your local host port 8080. To open the app, go to http://localhost:8080.
+
+![Services list]({{site.url}}{{site.baseurl}}/images/trace-analytics/sample-app.png)
+
+In the sample app, Hot R.O.D., select any one of the buttons to generate data. Now you can view trace data in Dashboards.
+
+### Step 4: View trace data in OpenSearch Dashboards
+
+After you generate Jaeger trace data you can go to OpenSearch Dashboards to view your trace data.
+
+Go to Dashboards **Trace analytics** at [http://localhost:5601/app/observability-dashboards#/trace_analytics/home](http://localhost:5601/app/observability-dashboards#/trace_analytics/home).
+
+## Use trace analytics in OpenSearch Dashboards
+
+To analyze the Jaeger trace data in Dashboards, first set up the trace analytics functionality. To get started, see [Get started with trace analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/get-started/).
+
+### Data sources
+
+You can specify either Data Prepper or Jaeger as the data source when you perform trace analytics.
+From Dashboards, go to **Observability > Trace analytics** and select Jaeger.
+
+![Select data source]({{site.url}}{{site.baseurl}}/images/trace-analytics/select-data.png)
+
+## Dashboard view
+
+After you select Jaeger for the data source, you can view all of the indexed data in **Dashboard** view, including **Error rate** and **Throughput**.
+
+### Error rate
+
+You can view the trace error count over time in the **Dashboard** view and also see the top five combinations of services and operations that have a non-zero error rate.
+
+![Error rate]({{site.url}}{{site.baseurl}}/images/trace-analytics/error-rate.png)
+
+### Throughput
+
+With **Throughput** selected, you can see the throughput of traces on Jaeger indexes that are coming in over time.
+
+You can select an individual trace from **Top 5 Service and Operation Latency** list and view the detailed trace data.
+
+![Throughput]({{site.url}}{{site.baseurl}}/images/trace-analytics/throughput.png)
+
+You can also see the combinations of services and operations that have the highest latency.
+
+If you select one of the entries for Service and Operation Name and go to the **Traces** column to select a trace, it will automatically add the service and operation as filters.
+
+## Traces
+
+In **Traces**, you can see the latency and errors for the filtered service and operation for each individual Trace ID in the list.
+
+![Select data source]({{site.url}}{{site.baseurl}}/images/trace-analytics/service-trace-data.png)
+
+If you select an individual Trace ID, you can see more detailed information about the trace, such as time spent by the service and each span for the service and operation. You can also view the payload that you get from the index in JSON format.
```

Comment:
Same comment re: "Trace ID". "time spent by the service and each span for the service and operation" is a bit too muddled/unclear. Please simplify/clarify and tag me to review. Can "that you get from" just be something like "returned by", "provided by", or "received from"?

Reply:
thanks! I clarified by indicating UI elements in bold font.

Reply:
hi @natebower, could you please verify this change? Thanks

"If you select an individual trace ID, you can see more detailed information about the trace, such as **Time spent by the service** and **Spans**. You can also view the index payload in JSON format."

Reply:
@alicejw-aws LGTM